CleverFerret/src/main/java/com/universalmedialibrary/services/ai/EnhancedAIContentCacheService.kt

```
@@ -456,12 +458,91 @@ class EnhancedAIContentCacheService @Inject constructor(
+    }
+    
+    /**
+     * Clear old cache (items older than 30 days)
+     */
+    suspend fun clearOldCache(): Boolean = withContext(Dispatchers.IO) {
+        try {
+            val cutoff = System.currentTimeMillis() - (30 * 24 * 60 * 60 * 1000L)
+            val oldItems = _cacheItems.value.filter { it.createdAt < cutoff }
+            oldItems.forEach { deleteCacheItem(it.id) }
+            true
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)